jwst/ami/tests/test_ami_interface.py

```diff
@@ -7,28 +7,29 @@
 from jwst.ami import AmiAnalyzeStep
 
 
-def test_ami_analyze_cube_fail():
-    """Make sure ami_analyze fails if input is CubeModel (_calints)"""
+@pytest.fixture()
+def example_model():
     model = datamodels.CubeModel((25, 19, 19))
     model.meta.instrument.name = "NIRISS"
     model.meta.instrument.filter = "F277W"
     model.meta.observation.date = "2019-01-01"
     model.meta.observation.time = "00:00:00"
-    with pytest.raises(RuntimeError):
-        AmiAnalyzeStep.call(model)
+    return model
 
 
-def test_ami_analyze_no_reffile_fail(monkeypatch):
+@pytest.mark.parametrize("oversample", [2, 4])
```

Comment:
I just picked 2 "random" even numbers here. Obviously we can't test them all but we might consider what a user might try to input.